<launch>

	 <set_env name="RCUTILS_CONSOLE_OUTPUT_FORMAT" value="{function_name}(): {message} "/>
	 <set_env name="RCUTILS_COLORIZED_OUTPUT" value="1"/>

    <!-- 変数 -->
    <arg name="name_space" default="ns1"/>
    <arg name="pkg_name" default="dynamixel_handler"/>
    
    <node pkg="$(var pkg_name)" exec="dynamixel_unify_baudrate_node" name="dxl_unify_handler" namespace="$(var name_space)" output="screen">
        <param from="$(find-pkg-share $(var pkg_name))/config/dynamixel_unify_baudrate.yaml"/>
    </node>
</launch>